specifyweb/frontend/js_src/lib/components/Header/index.tsx

```diff
@@ -194,9 +206,21 @@ export function MenuButton({
   readonly props?: TagProps<'a'> & TagProps<'button'>;
 }): JSX.Element | null {
   const [position] = userPreferences.use('header', 'appearance', 'position');
+  const [isSideBarLight] = userPreferences.use(
+    'general',
+    'appearance',
+    'lightSideBarBackground'
+  );
+  const isDarkMode = useDarkMode();
   const getClassName = (isActive: boolean): string => `
     p-[1.4vh]
-    ${isActive ? 'bg-brand-300 !text-white' : 'text-white'}
+    ${
+      isActive
+        ? 'bg-brand-300 text-white'
+        : isDarkMode || isSideBarLight === 'dark'
```

Comment:
just like above you created `isMenuLight` variable, could you do the same here?

(optional) (or create a hook like I suggested in https://github.com/specify/specify7/pull/4163/files#r1375147892 to have it consistent, and not duplicate)

Reply:
this one was clear for me without having to create a variable...

Reply:
oh, I didn't notice that the expressions are slightly different. sorry about that.
then yeah, separate variable is not necessary, but having it does not hurt